src/config.rs

```diff
@@ -182,6 +184,9 @@ impl Default for OutputConfig {
         let editor_command = vec![String::from("vim"), String::from("-p")];
         let editor_batch = true;
         let browser_command = vec![String::from("google-chrome-stable")];
+        let mut label_colors = HashMap::new();
+        label_colors.insert(String::from("read"), String::from("Green"));
+        label_colors.insert(String::from("current"), String::from("Yellow"));
```

Comment:
I would like to be a bit extra-nitpicky when I set defaults. The way the `set` command is written for labels is:

```console
$ ls zeus | set is good
```

Hence labels read like adjectives. However, `read` is not an adjective (Well, there is an adjective form but it doesn't mean that some material's reading process has been completed). Therefore I would prefer `done` as a default.

`current` is surely and adjective, but instead of meaning that something is happening, it's meaning is rather closer to 'present-day' or 'contemporary'. Therefore I would prefer `active` or even `in_progress` as a default.

Please also feel free to suggest better default words for `label_colors`.

Reply:
I think `active` would be better as it is less verbose for users to write/edit in their config.